src/ws/client.rs

```diff
@@ -541,15 +635,68 @@ impl ClientWriter {
     /// Send text frame
     #[inline]
     pub fn text<T: Into<Binary>>(&mut self, text: T) {
+        assert!(
```

Comment:
Is it likely to happen for user?
Maybe it would be better to have `debug_assert` and print warning in release mode instead of sending frame?

I'm ok with panic in `*_part` methods though as it is mis-use